moveit_core/CMakeLists.txt

```diff
@@ -158,35 +157,36 @@ message(STATUS " *** Building MoveIt ${MOVEIT_VERSION} ***")
 configure_file("version/version.h.in" "${VERSION_FILE_PATH}/moveit/version.h")
 install(FILES "${VERSION_FILE_PATH}/moveit/version.h" DESTINATION include/moveit)
 
-add_subdirectory(version)
-add_subdirectory(macros)
+add_subdirectory(background_processing)
 add_subdirectory(backtrace)
+add_subdirectory(collision_distance_field)
+add_subdirectory(constraint_samplers)
+add_subdirectory(controller_manager)
+add_subdirectory(distance_field)
+add_subdirectory(dynamics_solver)
 add_subdirectory(exceptions)
+add_subdirectory(kinematics_base)
+add_subdirectory(kinematic_constraints)
+add_subdirectory(kinematics_metrics)
+add_subdirectory(macros)
+add_subdirectory(planning_interface)
+add_subdirectory(planning_request_adapter)
+add_subdirectory(planning_scene)
 add_subdirectory(profiler)
-add_subdirectory(utils)
-add_subdirectory(background_processing)
 add_subdirectory(robot_model)
-add_subdirectory(collision_detection)
-add_subdirectory(kinematics_base)
-add_subdirectory(transforms)
 add_subdirectory(robot_state)
-add_subdirectory(collision_detection_fcl)
 add_subdirectory(robot_trajectory)
-add_subdirectory(kinematic_constraints)
-add_subdirectory(trajectory_processing)
-add_subdirectory(planning_scene)
-add_subdirectory(controller_manager)
 add_subdirectory(sensor_manager)
-add_subdirectory(constraint_samplers)
-add_subdirectory(planning_interface)
-add_subdirectory(planning_request_adapter)
-add_subdirectory(distance_field)
-add_subdirectory(collision_distance_field)
-add_subdirectory(kinematics_metrics)
-add_subdirectory(dynamics_solver)
-add_subdirectory(collision_detection_bullet)
+add_subdirectory(online_signal_smoothing)
+add_subdirectory(trajectory_processing)
+add_subdirectory(transforms)
+add_subdirectory(utils)
+add_subdirectory(version)
 
-pluginlib_export_plugin_description_file(moveit_core collision_detector_bullet_description.xml)
+# TODO(andyz): why don't these compile when alphabetized with the rest?
```

Comment:
Probably missing dependencies. For instance, `collision_detection` calls `ament_add_gtest` without calling `find_package(ament_cmake_gtest)`... I don't think it belongs into this PR, though.